styles/style.css

```diff
@@ -0,0 +1,229 @@
+@import url("https://fonts.googleapis.com/css2?family=Ubuntu:wght@400;500;700&display=swap");
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  font-family: "Ubuntu", sans-serif;
+}
+
+section {
+  padding: 100px 0;
+}
+.max-width {
+  padding: 0 80px;
+  margin: auto;
+}
+
+.about .about-content,
+.portfolio .portfolio-content,
+.skills .skills-content,
+.contact .contact-content {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-between;
+  font-size: 15px;
+  line-height: 1.5;
+}
+section .title {
+  position: relative;
+  text-align: center;
+  font-size: 40px;
+  font-weight: 500;
+  margin-bottom: 60px;
+  padding-bottom: 20px;
+}
+
+/* navbar */
+.navbar {
+  position: fixed;
+  width: 100%;
+  z-index: 999;
+  padding: 30px 0;
+  background: white;
+  box-shadow: 0 2px 5px 0 rgb(0 0 0 / 16%), 0 2px 10px 0 rgb(0 0 0 / 12%);
+}
+
+.navbar .max-width {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+.navbar .logo a {
+  color: #000;
+  font-size: 35px;
+  font-weight: 600;
+  letter-spacing: 4px;
+}
+.navbar .logo a span {
+  color: rgb(235, 74, 106);
+}
+
+.navbar .menu li {
+  list-style: none;
+  display: inline-block;
+}
+.navbar .menu li a {
+  display: block;
+  color: #000;
+  font-size: 18px;
+  font-weight: 500;
+  margin-left: 25px;
+  letter-spacing: 4px;
+}
+.navbar .menu li a:hover {
+  color: rgb(235, 74, 106);
+}
+
+/* menu btn */
+.menu-btn {
+  display: none;
+}
+
+/* home section */
+.home {
+  display: flex;
+  height: 100vh;
+  color: #fff;
+  min-height: 500px;
+  /* background-image: url(/assets/buildings.jpeg); */
+  background-image: url(https://source.unsplash.com/1600x900/?Architecture);
+  background-size: cover;
+  background-attachment: fixed;
+}
+.home .max-width {
+  width: 100%;
+}
+
+.home .home-content .text-1 {
+  font-size: 40px;
+  font-weight: 600;
+  position: absolute;
+  background: #000000d0;
+  margin-top: -3rem;
+}
+.home .home-content .text-2 {
+  margin: 5px 0;
+  position: absolute;
+  font-size: 75px;
+  color: rgb(235, 74, 106);
+  padding: 0 5px;
+  background: #fff;
+}
+
+/* about section */
+
+.about .about-content {
+  display: grid;
+  grid-template-columns: min-content 1fr;
```

Comment:
Because you only have one column for your entire page, it's not necessary to use grid. But good work finding a way to practice using CSS grid.